src/pages/History/History.tsx

```diff
@@ -18,15 +20,12 @@ import {
 import { numberWithCommas } from '@services/UsefulMethods/UIMethods';
 import IHistoryItem from '@models/IHistoryItem';
 import { useGetUserHistoryQuery } from '@store/Controllers/UserController/UserController';
-import SmallModal from '@components/ModalWindows/SmallModal/SmallModal';
-import ConfirmationModal from '@components/ModalWindows/ConfirtmationModal/ConfirmationModal';
-import { useDeleteExpenseByGroupMutation } from '@store/Controllers/ExpensesController/ExpensesController';
-import ExpenseModal from '@components/ModalWindows/ExpenseModal/ExpenseModal';
-import { Omiter } from '@services/UsefulMethods/ObjectMethods';
+import { useAppSelector } from '@hooks/storeHooks/useAppStore';
 
 interface IColumnsHistory extends IHistoryItem {edit_remove?: string}
 
 const History: React.FC = () => {
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency);
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)